README.md

```diff
@@ -1,8 +1,42 @@
 # BitClout Frontend
 This is the repository for the reference BitClout Angular app. It serves as an
 interface that allows users to interact with the data on the BitClout blockchain.
+## Getting started
+Bitclout frontend-ui depends on node Node.js, currently tested on v14.16.1.
+
+On MacOs  and Unbuntu, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/creationix/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
+
+To install NVM for OSX/Linux, add the following command in your terminal:
+
+```
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash
```

Comment:
This seems like a really terrible idea blindly running shell scripts from unofficial urls.  nvm repo is here: https://github.com/nvm-sh/nvm

Reply:
sorry honest mistake will correct it asap with
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash

Reply:
added the official url